sdk/python/sawtooth_sdk/processor/state.py

```diff
@@ -90,3 +90,31 @@ def set(self, entries, timeout=None):
             raise InvalidTransaction(
                 'Tried to set unauthorized address: {}'.format(addresses))
         return response.addresses
+
+    def delete(self, entries, timeout=None):
+        """
+        delete an address in the validator's merkle state
+        Args:
+            entries (list): list of StateEntry
+            timeout: optional timeout, in seconds
+
+        Returns:
+            addresses (list): a list of addresses that were deleted
```

Comment:
Following the pattern for gets and sets which do the same. Changing the pattern of the sdk state handling is beyond the scope of this PR.